### PR TITLE
chore: clean up README file

### PR DESCRIPTION
**Pull request by MakMuftic**
Remove unnecessary/outdated information from the README file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,14 +31,3 @@ Requires `nodejs` and `npm`.
...
-# ChainSafe Security Policy
-
-## Reporting a Security Bug
-
-We take all security issues seriously, if you believe you have found a security issue within a ChainSafe
-project please notify us immediately. If an issue is confirmed, we will take all necessary precautions
-to ensure a statement and patch release is made in a timely manner.
-
-Please email us a description of the flaw and any related information (e.g. reproduction steps, version) to
-[security at chainsafe dot io](mailto:security@chainsafe.io).
```
Comment (MakMuftic) [inline README.md:46]: Actually @nmlinaric has a good idea for this, we should revert these changes but use sygma dev mail. @P1sar what do you say about this? We can also create security@buildwithsygma.com and use that?

Comment (P1sar) [reply README.md:46]: Yes, sounds good, I will ask to create this email.

Comment (P1sar) [reply README.md:46]: we can leave this [dev@buildwithsygma.com](mailto:dev@buildwithsygma.com) for now

**Review by nmlinaric — APPROVED** [review]

**Review by mpetrun5 — APPROVED** [review]
